src/app/locationForm/LocationForm.jsx

```diff
@@ -73,7 +96,13 @@ class LocationForm extends Component {
           <div className="container">
             <div className="row px-4">{this.routeComponents}</div>
           </div>
-          <div className="position-absolute" style={{ right: 0, bottom: 12 }}>
+          <div 
+            className="position-absolute" 
+            style={{ 
+             right: 0,
+             bottom: 12,
+             display: this.state && this.state.inputFocused ? 'none' : 'block' 
```

Comment:
I didn't realize the arrows should be hidden on input focus. Does that apply to every page?

Reply:
This is to work around a behavior in Android, where the open keyboard reduces the page size. Because the buttons are absolutely positioned, they stick to the bottom of the resized page, which looks like this:

<img src="https://photos.google.com/share/AF1QipPoPUhauVuf1s8a7ljmapvdqZTtm4tXi8ijbdz0xxS8mRk1AXh4fNHS7hjgu7imWw/photo/AF1QipMKyqG6_6cGqS_Dxx0K3j_1GalzBhgrg_JywSq_?key=dVltMUF5Vi1KX0ZSRXNEZUR1NWg1VmVHWEhsbWRR"></img>

See row 16 on the master bug list.